src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -195,6 +195,8 @@ $ cwltool --version
 /usr/local/bin/cwltool ${this.cwltoolVersion}
 \`\`\`
 
+If you have issues installing cwltool using the above method try running \`pip3 install cwltool\`, which will install the latest released version from PyPi that is compatible with your Python version.
```

Comment:
Should probably say something like "Although Dockstore has not tested with the latest released version from PyPi, if you have issues installing cwltool using the above method ..."